### ISSUE TITLE
[BUG] Node stuck in Removing status on Rancher UI Cluster Manager

### ISSUE THREAD

**Issue by quoc9x** [rancher/rancher#39743]
**Rancher Server Setup**
- Rancher version: v2.6.3-patch1
- Installation option (Docker install/Helm Chart): Docker install

**Information about the Cluster**
- Kubernetes version: v1.20.11
- Cluster Type (Local/Downstream): Local
* Custom = Running a docker command on a node

**User Information**
- What is the role of the user logged in? Admin

**Describe the bug**
When I delete a node k8s cluster from Rancher UI Cluster Manager, it is stuck in `Unavailable` status.
Then, I delete that node from Rancher UI Cluster Explore, it's successfully.       

![image](https://user-images.githubusercontent.com/67582708/204773340-32318766-5a2a-4ebc-9cc2-ebe0c92d05b3.png)

I use `kubectl get nodes` the node is also deleted.   

![image](https://user-images.githubusercontent.com/67582708/204778441-595c803d-6b28-4bf3-9f1b-67e35da122f4.png)

But at Rancher UI Cluster Manager that node stuck in `Removing` status forever.   

![image](https://user-images.githubusercontent.com/67582708/204778641-24a46e50-af46-4745-9cb3-1f802908c195.png)

Is there any way to delete that node from Rancher UI Cluster Manager?

Thanks!

**Comment by jacobfra** [comment]
You could try and remove the machine object that rancher users and see if that cleans it up.

**Comment by quoc9x** [comment]
> You could try and remove the machine object that rancher users and see if that cleans it up.

Delete the rancher and reinstall it?

**Comment by jacobfra** [comment]
> > You could try and remove the machine object that rancher users and see if that cleans it up.
> 
> Delete the rancher and reinstall it?

No,  rancher is an application running in a cluster (sometimes it is called local).  rancher uses CRD's to define VMs and MachineIds and maps them to a Cluster CRD object. 

you could try and remove those objects from the manager cluster (local).
$ kubectl get nodes.management.cattle.io -n <clusterid>
...  shows a list of machines attached to a cluster....

$ kubectl get clusters.management.cattle.io <clusterid>
... shows the machine/vms rancher things is in the cluster object shown in UI...

NOTE:  I do not work or support rancher code.. I am just a user... so try at your own risk.

**Comment by quoc9x** [comment]
I tried that but the command itself got stuck.

![image](https://user-images.githubusercontent.com/67582708/205417706-a7b86e53-16c2-45cc-90a1-395ad0181cba.png)
